dependency-reduced-pom.xml

```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this meant to be included? Usually I see this (or similar) file generated?

Reply:
now removed - should we add this file to '.gitignore' because it gets generated by the build and can easily be added to commits by mistake?

Reply:
@pjfanning Yes, makes sense to add to `.gitignore`